chipsec/modules/common/memconfig.py

```diff
@@ -93,18 +93,20 @@ def check_memmap_locks(self):
             self.logger.log('[*] Checking legacy register lock state:')
         else:
             self.logger.log('[*] Checking register lock state:')
-        for r in regs:
-            if not self.cs.is_register_defined(r) or not self.cs.register_has_field(r, memmap_registers[r]):
-                self.logger.log_important('Skipping Validation: Register {} or field {} was not defined for this platform.'.format(r, memmap_registers[r]))
+        for reg in regs:
+            reg_field = memmap_registers[reg]
+            if not self.cs.register_has_field(reg, reg_field):
+                self.logger.log_important(f'Skipping Validation: Register {reg} or field {reg_field} was not defined for this platform.')
```

Comment:
```suggestion
                self.logger.log_important(f"Skipping Validation: Register {reg} or field {reg_field} was not defined for this platform.")
```
We should be consistent, but I think we should go with double quotes. `"`